From: Stefan Haberland <sth@linux.ibm.com>
Subject: dasd: Add IFCC notice message
Patch-mainline: no, SLES 11 only
References: bnc#1104481, LTC#170484

Description:  dasd: Add IFCC notice message
Symptom:      The IFCC warning message was moved to the s390dbf for
              debugging purposes and to not distract customer with
              seldom IFCC events that likely have no impact.
Problem:      Some critical customers have additional monitoring tools
              that rely on the IFCC message.
Solution:     Add the IFCC message again with notice level.
              In normal operation these messages are hidden but
              customers have the ability to enable it by lowering the
              syslog severity.
Reproduction: Have a system with IFCC events. 


Signed-off-by: Stefan Haberland <sth@linux.ibm.com>
Acked-by: Johannes Thumshirn <jthumshirn@suse.de>
---
 drivers/s390/block/dasd_3990_erp.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/s390/block/dasd_3990_erp.c
+++ b/drivers/s390/block/dasd_3990_erp.c
@@ -2226,6 +2226,8 @@ dasd_3990_erp_control_check(struct dasd_
 					   | SCHN_STAT_CHN_CTRL_CHK)) {
 		DBF_DEV_EVENT(DBF_WARNING, device, "%s",
 			    "channel or interface control check");
+		dev_notice(&device->cdev->dev,
+			   "channel or interface control check\n");
 		erp = dasd_3990_erp_action_4(erp, NULL);
 	}
 	return erp;
